src/category_theory/model/category.lean

```diff
@@ -19,14 +19,75 @@ structure is_model_category (W C F : morphism_class M) : Prop :=
 -- TODO: Show that it follows that W is closed under retracts. See
 -- https://ncatlab.org/nlab/show/model+category#ClosureOfMorphisms
 
+lemma is_model_category.weq_of_weq_retract_fib { W C F : morphism_class M } ( h : is_model_category W C F )
```

Comment:
So I have a question about this. Trying to rephrase this to use `model_category.{v} M` rather than `is_model_category W C F` (and omitting \MCM from the context) I end up with 
```
lemma weq_of_weq_retract_fib [ h : model_category.{v} M ]
 {a b a' b': M} {f : a ⟶ b} {f' : a' ⟶ b'} (r : retract f f') (hf : h.W f) (hf': h.F f') : h.W f'
```
but `h.W` and `h.F` produce the following error:

"invalid field notation, function 'category_theory.model_category.W' does not have explicit argument with type (category_theory.model_category ...)".

I don't understand this - if I `#check h.W`, it's a morphism class on M like I expect.